README.md

```diff
@@ -8,12 +8,23 @@ responding to events and interactive buttons.
 
 ## Running locally
 
+### 0. Create a new Slack App
+
+- Go to https://api.slack.com/apps
+- Click **Create App**
+- Choose a workspace
+- Enter App Manifest using contents of `manifest.yaml`
```

Comment:
I know we opted for JSON in the template apps we created that make use of the manifest. Do we want to continue that trend for this? cc: @mwbrooks @stevengill

Reply:
My intuition is that bolt-js users will probably prefer JSON over YAML but that's just my speculation.